### PR TITLE
HW 1 - Gabrielle Bogert

### PR DESCRIPTION
**Pull request by francisco-bogert**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+* Digital media and communication
+* Public transportation (yes I'm a NUMTOT)
+* Libraries
+* Listening to, sharing, and discovering music
+* Scrolling through Twitter and TikTok for hours
+* Going down Wikipedia rabbit holes from the hours of 1 to 4 AM each night of quarantine
+* Reinventing myself constantly by adding hobbies and skills to the institutional memory of my physical form...we're currently on baking
+
+*The last thing that made me laugh is*
+![Funny Tweet](Desktop/hahaha.png)
```
Comment (deblnia) [inline README.md:17]: If you don't commit the image (hahaha.png) relative paths like this won't render on Github.

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+
+*The latest thing I've baked is*
+[Earl Grey Yogurt Cake](https://www.bonappetit.com/recipe/earl-grey-yogurt-cake)
+
+
+### PROCESS
```
Comment (deblnia) [inline README.md:23]: For future homeworks, put your process comments in the Pull Request text box itself. 😄

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
...
+
+**These were my steps to get changes into README.md on GitHub** 
+1. Forked the hw01 repository
+2. Cloned the fork onto my computer by creating a version control project
+3. Made changes to the README.md by adding my biography and homework process
+4. Committed my changes in R Studio
+5. Pushed my changes to GitHub
+6. Made a pull request to submit my homework
+
+I don't know how often I should be committing changes, for this assignment I could do this in one sitting so I only committed once. But let me know if I should be committing more often, based on X amount of lines of code written or X amount of time spent working.
```
Comment (deblnia) [inline README.md:34]: A good rule of thumb is that you want to commit just as often as you'd save -- basically every time to do some work that you'd consider submit-able. Commiting once for an assignment like this is 100% fine.

**Review by deblnia — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     X      |              |            |
| **Coding strategy**         |    X       |              |            |
| **Presentation**            |           |    X          |            |
| **Achievement, creativity** |      X     |              |            |
| **Reproducibility**         |       X    |              |            |

Remarks:

* Overall, great work! 💯 
* I docked presentation points for the un-rendered image. [This ](https://www.earthdatascience.org/courses/earth-analytics/document-your-science/add-images-to-rmarkdown-report/) might help in understanding why that happened.

**Review by paogomco — COMMENTED** [review]
Hi Gabrielle, 
Wins: 
Excellent use of the headers, list and also the link you attached,
Needs work:
You may haven't committed the picture.

**Review by whitneybeamer — COMMENTED** [review]
Hi Gabrielle,
This was a really great execution of the assignment! 
Loved the fact that you incorporated list and just a lot of details about yourself! 
I also really enjoyed the funny tweet you incorporated at the end :)
One way to improve this, which I am sure you have learned by now, is committing images to git hub as well as commenting your "assignment review" under the pull request. 
Other wise great work, as I would give all excellents!

**Pull request closed without merging**